packages/cli/test/integration/credentials.api.test.ts

```diff
@@ -2,6 +2,7 @@ import express from 'express';
 import { UserSettings } from 'n8n-core';
 import { Db } from '../../src';
 import { randomName, randomString } from './shared/random';
+import config from '../../config';
```

Comment:
Separate PR https://github.com/n8n-io/n8n/pull/3601 introduces `authAgent`, can also be used to reduce boilerplate here.